VERSION

```diff
@@ -1 +1 @@
-1.0.1
+2.0.0
```

Comment:
Is this a breaking change?

Reply:
Yes I'd say so although it's not black and white, `Bump major version due to breaking package and node version updates.`, we upgrade the packages which require a newer node version.